### PR TITLE
Docs: add content outline to Cloud what's new

### PR DESCRIPTION
**Pull request by imatwawana**
to do: need docs link

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -18,6 +18,19 @@ weight: -37
...
+## Content outline
+
+<!-- Thanos Karachalios -->
+<!-- OSS,Enterprise -->
+
+October 12, 2023
+
+_Generally Available in Grafana Cloud_
+
+Introducing Content Outline in Grafana Explore. We recognized the challenges of complex mixed queries, as well as, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](), as well as the following video demo.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:30]: Generally we don't capitalize features like this that don't show up in the UI with the exact same name, but I'll leave it for now. Something to keep in mind as you're naming features.

Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:30]:
```suggestion
Introducing Content Outline in Grafana **Explore**. We recognized the challenges of complex mixed queries, as well as, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](), as well as the following video demo.
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -18,6 +18,19 @@ weight: -37
...
+## Content outline
+
+<!-- Thanos Karachalios -->
+<!-- OSS,Enterprise -->
+
+October 12, 2023
+
+_Generally Available in Grafana Cloud_
+
+Introducing Content Outline in Grafana **Explore**. We recognized the challenges of complex mixed queries, as well as, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/latest/explore/) (See Content outline section), as well as the following video demo.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:30]:
```suggestion
Introducing Content Outline in Grafana **Explore**. We recognized the challenges of complex mixed queries, as well as, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/) (See Content outline section), as well as the following video demo.
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 3]
```diff
@@ -18,6 +18,19 @@ weight: -37
...
+## Content outline
+
+<!-- Thanos Karachalios -->
+<!-- OSS,Enterprise -->
+
+October 12, 2023
+
+_Generally Available in Grafana Cloud_
+
+Introducing Content Outline in Grafana **Explore**. We recognized the challenges of complex mixed queries, as well as, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/) (See Content outline section), as well as the following video demo.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:30]:
```suggestion
Introducing Content Outline in Grafana **Explore**. We recognized the challenges of complex mixed queries, as well as, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/#content-outline), as well as the following video demo.
```

**Review by Eve832 — APPROVED** [review]

**Pull request merged**